network/vpc/reference-content/vpc-migration.mdx

```diff
@@ -35,87 +35,83 @@ If you are unsure about some of the terminology in this document, refer to the s
-| I have a Private Network, attached to a Public Gateway, and DHCP is enabled on the gateway.   | DHCP **will** be automatically activated on your Private Network, and cannot be deactivated. We will take care of IP management of your resources within your Private Network (see details above). <br /><br/>All the Public Gateway's leases will be synchronized with the managed DHCP, so there is no impact for you and no action is necessary. |
-| I have a Private Network with my own DHCP server running on it | We assume that your network is therefore not attached to a Public Gateway with DHCP enabled (which would cause conflicts with your own DHCP server). DHCP will **not** be automatically activated on your Private Network, to avoid any conflicts with your existing IP addressing. <br /><br/>You will see a button in the Scaleway console inviting you to activate DHCP if/when you are ready. Note that activating DHCP is permanent, and there is no way to undo the activation, so proceed with caution. |
+| I have a Private Network, with no Public Gateway attached to it   | DHCP has **not** been automatically activated on your Private Network, to avoid any conflicts with your existing IP addressing. <br /><br/>You should see a button in the Scaleway console inviting you to activate DHCP if/when you are ready. Note that activating DHCP is permanent, and there is no way to undo the activation, so proceed with caution. |
+| I have a Private Network, attached to a Public Gateway, but DHCP is disabled on the gateway.   | DHCP has **not** be automatically activated on your Private Network, to avoid any conflicts with your existing IP addressing.  <br /><br/>You should see a button in the Scaleway console inviting you to activate DHCP if/when you are ready. Note that activating DHCP is permanent, and there is no way to undo the activation, so proceed with caution. |
+| I have a Private Network, attached to a Public Gateway, and DHCP is enabled on the gateway.   | DHCP **has** been automatically activated on your Private Network, and cannot be deactivated. We take care of IP management of your resources within your Private Network (see details above). <br /><br/>All the Public Gateway's leases are synchronized with the managed DHCP, so there is no impact for you and no action is necessary. |
+| I have a Private Network with my own DHCP server running on it | We assume that your network is therefore not attached to a Public Gateway with DHCP enabled (which would cause conflicts with your own DHCP server). DHCP has **not** beeb automatically activated on your Private Network, to avoid any conflicts with your existing IP addressing. <br /><br/>You should see a button in the Scaleway console inviting you to activate DHCP if/when you are ready. Note that activating DHCP is permanent, and there is no way to undo the activation, so proceed with caution. |
```

Comment:
```suggestion
| I have a Private Network with my own DHCP server running on it | We assume that your network is therefore not attached to a Public Gateway with DHCP enabled (which would cause conflicts with your own DHCP server). DHCP has **not** been automatically activated on your Private Network, to avoid any conflicts with your existing IP addressing. <br /><br/>You should see a button in the Scaleway console inviting you to activate DHCP if/when you are ready. Note that activating DHCP is permanent, and there is no way to undo the activation, so proceed with caution. |
```